internal/api/databases.go

```diff
@@ -17,79 +14,15 @@ func initDatabases(apiRouter *mux.Router, context *Context) {
 		return newContextHandler(context, handler)
 	}
 
+	// TODO: retire these endpoints
```

Comment:
It would probably be good to log a ticket for removing that alongside with updating any components that depend on this.